.github/workflows/bootstrap.yml

```
@@ -19,6 +19,10 @@ on:
     # nightly at 2:16 AM
     - cron: '16 2 * * *'
 
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.sha }}
```

I've been using `github.run_id` instead of `github.sha` as the fallback since `github.run_id` is guaranteed to be unique to every workflow run and `github.sha` could potentially exist on multiple branches.  Since we're already adding `github.workflow` to the group names you can get the same effect with `github.run_number` which is unique *per-workflow* for every run i.e.

```suggestion
  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.run_number }}
```